docs/contributor-guide.rst

```diff
@@ -135,3 +135,22 @@ Editor config
 
 - `python-black <https://atom.io/packages/python-black>`_
 - `linter-flake8 <https://atom.io/packages/linter-flake8>`_
+
+Overview of Resource and Collection interaction (from_json and from_data confusion) 
+-----------------------------------------------
+![diagram 1](/docs/resource:collectionRoute.png)
```

Comment:
@charlottemoremen can you provide a screenshot of the docs showing that this will render the 2 pictures properly?